homepage/homepage.go

```diff
@@ -135,28 +135,26 @@ func init() {
 		data:               zebedee.TimeseriesMainFigure{},
 		differenceInterval: mapper.PeriodYear,
 	}
-
 }
 
 func getLatestTimeSeriesData(ctx context.Context, zts []zebedee.TimeseriesMainFigure) zebedee.TimeseriesMainFigure {
 	var latest zebedee.TimeseriesMainFigure
-
-	for _, ts := range zts {
+	for i := 0; i < len(zts); i++ {
```

Comment:
Why change the `range` for loop to an. indexed one? I don't see anything wrong with the original?

Reply:
ok, The linter was complaining about this as while we are doing range each time it is getting a copy of the element rather than getting the element directly from the slice. So either we can make the it a slice of pointers or we can do the following
```
for index, _ := range zts
example := zts[index]
``` 
or
you can do the above option which i did, so that we will be getting the element directly from the slice. Hope this makes sense. 
```

Reply:
OK, that makes sense. 👍